packages/node/test/fixtures/63-nodejs-middleware/middleware.ts

```diff
@@ -0,0 +1,18 @@
+export const config = {
+  runtime: 'nodejs',
+};
+
+export default (req: Request) => {
+  const url = new URL(req.url);
+  const headers = new Headers({
+    'x-got-middleware': 'true',
+  });
+  if (url.pathname === '/' || url.pathname.startsWith('/api/')) {
+    // For `index.html` and `/api/node.js`, pass through
+    headers.set('x-middleware-next', '1');
```

Comment:
> For `index.html` and `/api/node.js`, pass through

What does that mean? Is this the only way to trigger this behavior? Ideally we don't need a next-specific header right?